Client build error: Target "slate-stress-test" links to target "nlohmann_json::nlohmann_json"

@sthapa 

Relating to #335, updating the `CMakeLists.txt` file has resulted in the following new error (see [action](https://github.com/slateci/slate-client-server/actions/runs/3519179273/jobs/5898877859)).
```shell
  cmake .. -DBUILD_CLIENT=True -DBUILD_SERVER=False -DBUILD_SERVER_TESTS=False -DSTATIC_CLIENT=True
  shell: sh -e {0}
  env:
    RELEASE_VERSION: 1.0.32-pre.20221122-001229
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Will build client
-- Found ZLIB: /lib/libz.so (found version "1.2.11") 
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.6.3") 
-- Found libcrypto:  (found version "1.1.1l")
-- Found ssl:  (found version "1.1.1l")
-- Found CURL: /usr/lib/libcurl.so (found version "7.79.1")  
Attempting fully static link
-- Configuring done
CMake Error at CMakeLists.txt:199 (add_executable):
  Target "slate-stress-test" links to target "nlohmann_json::nlohmann_json"
  but the target was not found.  Perhaps a find_package() call is missing for
  an IMPORTED target, or an ALIAS target is missing?


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
Error: Process completed with exit code 1.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -191,7 +191,6 @@ LIST(APPEND STRESS_TEST_SOURCES
         ${CMAKE_SOURCE_DIR}/src/KubeInterface.cpp
         ${CMAKE_SOURCE_DIR}/src/Process.cpp
         ${CMAKE_SOURCE_DIR}/src/Utilities.cpp
-        ${CMAKE_SOURCE_DIR}/src/Telemetry.cpp
         )
 SET(STRESS_TEST_COMPILE_OPTIONS
         -DSLATE_CLIENT
@@ -206,7 +205,6 @@ target_include_directories (slate-stress-test
         ${CURL_INCLUDE_DIRS}
         ${LIBCRYPTO_CFLAGS}
         ${SSL_CFLAGS}
-        ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
         $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}>
         )
 target_link_libraries(slate-stress-test
@@ -216,8 +214,6 @@ target_link_libraries(slate-stress-test
         ${LIBCRYPTO_LDFLAGS}
         z
         pthread
-        ${OPENTELEMETRY_CPP_LIBRARIES}
-        nlohmann_json::nlohmann_json
         )
 target_compile_options(slate-stress-test PRIVATE -DRAPIDJSON_HAS_STDSTRING -Os -std=c++11)
 
@@ -328,8 +324,8 @@ if(BUILD_SERVER)
             src/Archive.cpp
             src/FileHandle.cpp
             src/FileSystem.cpp
-            src/Process.cpp
-             src/Telemetry.cpp)
+            src/Process.cpp)
+#             src/Telemetry.cpp)
     target_include_directories (slate-test-database-server
             PUBLIC
             $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
@@ -341,7 +337,7 @@ if(BUILD_SERVER)
             ${CURL_INCLUDE_DIRS}
             ${LIBCRYPTO_CFLAGS}
             ${SSL_CFLAGS}
-            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+#            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
             )
     target_link_libraries(slate-test-database-server
             PUBLIC
@@ -351,7 +347,7 @@ if(BUILD_SERVER)
             ${Boost_LIBRARIES}
             ${ZLIB_LIBRARIES}
             pthread
-            ${OPENTELEMETRY_CPP_LIBRARIES}
+#            ${OPENTELEMETRY_CPP_LIBRARIES}
             )
 
     add_executable(slate-test-helm-server
@@ -368,7 +364,7 @@ if(BUILD_SERVER)
             ${CURL_INCLUDE_DIRS}
             ${LIBCRYPTO_CFLAGS}
             ${SSL_CFLAGS}
-            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+#            ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
             )
     target_compile_options(slate-test-helm-server PRIVATE ${YAMLCPP_CFLAGS} )
     target_link_libraries(slate-test-helm-server
@@ -380,7 +376,7 @@ if(BUILD_SERVER)
             ${Boost_LIBRARIES}
             ${ZLIB_LIBRARIES}
             pthread
-            ${OPENTELEMETRY_CPP_LIBRARIES}
+#            ${OPENTELEMETRY_CPP_LIBRARIES}
             )
 
     file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/tests")

